src/de/communities.md

```diff
@@ -0,0 +1,121 @@
+
+Der nächste Schritt ist das Erstellen des Logos deiner Community. Die ideale Größe beträgt 256x256 Pixel aber falls du hineinzoomen möchtest dann solltest du ein höher aufgelöstes Bild verwenden. Siehe [Logo](#logo) für weitere Informationen zur Anpassung deines Logos.
+
+Nun kannst du die Grundfarbe deiner Community wählen. Transparente Teile deines Logos werden mit dieser Farbe gefüllt. Sie wird außerdem an verschiedenen Stellen verwendet wie z.B. als Hintergrundfarbe auf der Community-Profilseite oder als Community-Farbe um Mitglieder verschiedener Communities innerhalb einer Slotliste zu unterscheiden.
+
+Der letzte Schritt ist das auswählen/bestätigen deines Community-Handle. Dieser ist eine aus max. 32 Web-freundlichen Zeichen bestehende Zeichenkette die überwiegend in URLs verwendet wird. Halte das Handle einfach und kurz. Da diese Zeichenkette einmalig sein muss, werden wir dich informieren falls sie bereits von einer anderen Community verwendet wird.
+
+![Workflow Community erstellen](../images/communities/create-community-workflow.webp "Workflow Community erstellen")
+
+Schlussendlich wirst du auf die Profilseite deiner Community weitergeleitet. Hier kannst du alles ändern, eine Vorstellung hinzufügen und deine Community verwalten. Ein Einladungs-Link wird automatisch für dich erstellt um ihn mit deinen Community-Mitgliedern zu teilen. Siehe [Mitglieder einladen](#mitglieder-einladen) für weitere Informationen zu Mitgliedereinladungen.
+
+> [!IMPORTANT]
+> Du brauchst eine Einladung (Link) um einer Community beizutreten. Es gibt keinen anderen Weg für den Beitritt.
+
+![Gerade erstellte Community](../images/communities/fresh-community.webp "Gerade erstellte Community")
+
+## Mitglieder einladen
+
+![Überblick Einladungen](../images/communities/invites-overview.webp "Überblick Einladungen")
+
+Spieler können deiner Community nur mit einer gültigen Einladung betreten. Eine Einladung ist ein Link, kombiniert mit einem Ablaufdatum und einer Höchstanzahl an Verwendungen. Standardmäßig ist eine Einladung für 7 Tage gültig und unbegrenzt nutzbar. Du kannst diese Einstellungen ändern durch klicken auf **Einladungs-Link bearbeiten** in dem Fenster das erscheint wenn du auf **+ Einladung erstellen** klickst.
+
+![Einladungs-Links bearbeiten](../images/communities/edit-invite-links.webp "Einladungs-Links bearbeiten")
+
+Die verbleibende Zeit und die Anzahl der Verwendungen wird in der Einladungsübersicht angezeigt. Über das Drei-Punkte-Symbol kannst du das plattformabhängige **Teilen**-Menü öffnen, eine aktive Einladung **löschen** oder **Link kopieren** benutzen. Außerdem kannst du den Link schnell kopieren indem du auf den Einladungs-Code klickst.
+
+Sende nun den erstellten Link über deine bevorzugte Methode an deine Community-Mitglieder. Das Format eines Einladungslinks ist stets `https://arma.events/[EINLADUNGS-CODE]` also z.B. `https://arma.events/1YOtETcO`
```

Comment:
```suggestion
Sende nun den erstellten Link über deine bevorzugte Methode an deine Community-Mitglieder. Das Format eines Einladungslinks ist stets `https://arma.events/i/[EINLADUNGS-CODE]` also z.B. `https://arma.events/i/1YOtETcO`
```

Reply:
Roger. Will change that in both languages.